### ISSUE TITLE
Ubuntu + OSX Support?

### ISSUE THREAD

**Issue by greyghosttv** [pRizz/iota-transaction-spammer-webapp#11]
Sorry for such a vague question but is it possible to release Ubuntu and OSX support. I lots of shitty laptops just laying around and would love to help contribute to the tangle! Massive thumbs up

**Comment by pRizz** [comment]
Yep, that is on my to-do list. It will probably be in a different repo, which I'll link to.

**Comment by WyseNynja** [comment]
This is working for me on OSX with Chrome. What support is missing?

**Comment by pRizz** [comment]
I think it is broken on OSX Safari
